docs/charon/networking.md

```diff
@@ -81,11 +81,25 @@ All other charon protocols are only established between nodes in the same cluste
 In order for a charon client to connect to a relay, it needs the relay's [multiaddr](https://docs.libp2p.io/concepts/fundamentals/addressing/) (containing its public key and IP address).
 But a single multiaddr can only point to a single relay server which can easily be overloaded if too many clusters connect to it. Charon therefore supports resolving a relay’s multiaddr
 via HTTP GET request. Since charon also includes the unique `cluster-hash` header in this request, the relay provider can use
-[consistent header-based load-balancing](https://cloud.google.com/load-balancing/docs/https/traffic-management-global#traffic_steering_header-based_routing)to map clusters to one of many relays using a single HTTP address.
+[consistent header-based load-balancing](https://cloud.google.com/load-balancing/docs/https/traffic-management-global#traffic_steering_header-based_routing) to map clusters to one of many relays using a single HTTP address.
 
 The relay supports serving its runtime public multiaddrs via its `--http-address` flag.
 
 E.g., https://0.relay.obol.tech is actually a load-balancer that routes HTTP requests to one of many relays based on the `cluster-hash` header returning the target relay’s multiaddr
 which the charon client then uses to connect to that relay.
 
-The charon `--p2p-relays` flag therefore supports both multiaddrs as well as HTTP URls.       
\ No newline at end of file
+The charon `--p2p-relays` flag therefore supports both multiaddrs as well as HTTP URls.
+
+### Authenticating a distributed validator client
```

Comment:
there is already a "LibP2P Authentication and Security" section above, this feels duplicated.  And the context is more about the DKG process, it isn't about "authentication", rather move to a DKG section.

Reply:
I'm leaning towards deleting this section, because of duplication and dkg page already containing most of the details

Reply:
Yeah aligned, this section can go